chromium/rules.js

```diff
@@ -416,63 +430,70 @@ RuleSets.prototype = {
    */
   potentiallyApplicableRulesets: function(host) {
     // Have we cached this result? If so, return it!
-    var cached_item = this.ruleCache.get(host);
-    if (cached_item !== undefined) {
-      util.log(util.DBUG, "Ruleset cache hit for " + host + " items:" + cached_item.length);
+    if (this.ruleCache.has(host)) {
+      let cached_item = this.ruleCache.get(host);
+      util.log(DBUG, "Ruleset cache hit for " + host + " items:" + cached_item.size);
       return cached_item;
+    } else {
+      util.log(DBUG, "Ruleset cache miss for " + host);
     }
-    util.log(util.DBUG, "Ruleset cache miss for " + host);
```

Comment:
There are some issues with how this was merged, this line should have been kept (it should be `util.DBUG`).